playerctl/playerctl-daemon.c

```diff
@@ -332,9 +340,32 @@ static void context_remove_player(struct PlayerctldContext *ctx, struct Player *
     }
 }
 
+static void context_shift_active_player(struct PlayerctldContext *ctx) {
+    GError *error = NULL;
+    struct Player *p;
+
+    p = context_get_active_player(ctx);
```

Comment:
It would make me feel better if we had a guard here for no active player instead of relying on that guard in the method call.

Reply:
I will put you at ease :smile: